addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -238,7 +238,7 @@ def partners(self, country=None, grade=None, page=0, **post):
                 country = country_obj.search([('code', '=', request.geoip.country_code)], limit=1)
 
         # Group by country
-        country_domain = list(base_partner_domain)
+        country_domain = list(base_partner_domain) + [('country_id', '!=', False)]
```

Comment:
You should only add it inside the read_group directly as we reuse this domain to know how many partners there are in total (see `search_count`) below

Reply:
Oh, I see. My bad should be fixed now.